examples/ChatQnA/deploy/xeon.md

```diff
@@ -78,19 +72,33 @@ export https_proxy=${your_http_proxy}
 
 ## Prepare (Building / Pulling) Docker images
 
-This step will involve building/pulling ( maybe in future) relevant docker
+This step will involve building/pulling  relevant docker
 images with step-by-step process along with sanity check in the end. For
 ChatQnA, the following docker images will be needed: embedding, retriever,
 rerank, LLM and dataprep. Additionally, you will need to build docker images for
 ChatQnA megaservice, and UI (conversational React UI is optional). In total,
 there are 8 required and an optional docker images.
 
-The docker images needed to setup the example needs to be build local, however
-the images will be pushed to docker hub soon by Intel.
-
 ### Build/Pull Microservice images
 
-From within the `GenAIComps` folder
+::::{tab-set}
+
+:::{tab-item} Pull
+:sync: Pull
+
+Ifyou decide to pull the docker containers and not build them locally, 
+we can proceed to the next step where all the necessary containers will 
+be pulled in from dockerhub.
+
+:::
+:::{tab-item} Build
+:sync: Build
+
+From within the `GenAIComps` folder, checkout the release tag.
+```
+cd GenAIComps
+git checkout tags/v1.0
+```

```

Comment:
You didn't close this build tab-item or the enclosing tab-set.  After line 101 you need to do that by adding these two lines:
```
:::
::::
```

Reply:
no it does not end there, it ends much below (above use case setup)

Reply:
Oh my, nested tabs eh?  That can work but you'll need to add another colon to the fences of the containing elements.  

The inner-most tab-item uses three colons and their parent tab-set uses four colons.  (That's the normal case .) 

When a tab-set (with four colons) is nested within a tab-item, that parent tab-item must use five colons, and the parent tab-set of that uses six.  Clear?

Reply:
resolved.